### DISCUSSION TITLE
Multiple terms in LS analysis

### DISCUSSION THREAD

**Discussion by stefanocovino** [giordano/LombScargle.jl#37] [Q&A]
Dear friends,

I'm converting most of my Python-based code involving Lomb-Scargle analysis to Julia and this library works amazingly well. 

However, I see that, compared to the Python version, there is no implementation of the possibility to carry out the analysis with multiple Fourier terms. I see that you can get similar results defining LS models, yet I wonder if you are planning to add this functionality in a future. 

Thanks a lot for your attention,

    Stefano

**Comment by giordano** [comment]
No plans on my side, but a pull request implementing the desired feature would be welcome.
